app/Transformers/Solo/ScoreTransformer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace App\Transformers\Solo;
+
+use App\Models\Solo\Score;
+use App\Transformers\TransformerAbstract;
+
+class ScoreTransformer extends TransformerAbstract
```

Comment:
can you mind adding the `user` attribute? I think the client-side don't have user_id property atm
and possibly required to get scores by API v2 for future

+) suggestion: add `beatmap` attribute as availableIncludes (for getting scores list from single user etc)

Reply:
Are you sure you're in the right place? The client already knows the user and beatmap because that's what it used to... submit the score, no?

Reply:
I thought the Solo/ScoreTransformer to fetch additional score lists, But it could be fetched by Normal ScoreTransformer one (modifying some values). sorry